src/Schema/Quoter.php

```diff
@@ -204,6 +215,11 @@ public function quoteValue(mixed $value): mixed
         return "'" . str_replace("'", "''", addcslashes($value, "\000\032")) . "'";
     }
 
+    public function setTablePrefix(string $value): void
```

Comment:
Need add this method to QuoterInterface

Reply:
If add this to `QuoterInterface` it breaks BC.